DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: PSMatch
 Title: Handling and Managing Peptide Spectrum Matches
-Version: 1.13.2
+Version: 1.13.3
```

Comment:
No, this must stay at the current version as I haven't pushed to Bioconductor, and when I do, I set the new NEWS version to 'Nothing yet'. If unsure, check the Bioconductor devel version.